io.openems.edge.controller.ess.timeofusetariff/src/io/openems/edge/controller/ess/timeofusetariff/TimeOfUseTariffController.java

```diff
@@ -9,58 +9,63 @@
 import io.openems.edge.common.component.OpenemsComponent;
 import io.openems.edge.controller.api.Controller;
 
-public interface ControllerEssTimeOfUseTariffDischarge extends Controller, OpenemsComponent {
+public interface TimeOfUseTariffController extends Controller, OpenemsComponent {
 
 	public enum ChannelId implements io.openems.edge.common.channel.ChannelId {
-
 		/**
-		 * Current state of the Time of use tariff discharge controller.
+		 * Current state of the Time of use tariff controller.
 		 */
 		STATE_MACHINE(Doc.of(StateMachine.values()) //
+				.persistencePriority(PersistencePriority.VERY_HIGH) //
 				.text("Current state of the Controller")),
 
+		/**
+		 * Aggregated seconds when storage is being force charged.
+		 */
+		CHARGED_TIME(Doc.of(OpenemsType.LONG) //
+				.unit(Unit.CUMULATED_SECONDS) //
+				.persistencePriority(PersistencePriority.HIGH)), //
+		CHARGED(Doc.of(OpenemsType.BOOLEAN)//
+				.text("The controller currently recharge the battery")),
```

Comment:
`CHARGING` would be more correct here... but for backwards compatibility to be similar with `DELAYED` I guess it is fine.

The text is grammatically wrong

Isn't the STATE_MACHINE sufficient to serve all these `CHARGED`, `DELAYED`, etc. channels?